asapdiscovery-docking/asapdiscovery/docking/openeye.py

```diff
@@ -69,6 +72,18 @@ class POSITDocker(DockingBase):
         description="Allow retries with different options if docking fails initially",
     )
 
+    @root_validator
+    @classmethod
+    def omega_dense_check(cls, values):
+        """
+        Validate omega_dense
+        """
+        omega_dense = values.get("omega_dense")
+        use_omega = values.get("use_omega")
+        if omega_dense and not use_omega:
+            raise ValueError("Cannot use omega_dense without use_omega")
+        return values
+
```

Comment:
do we want to check this again in the CLI? I guess we want to avoid doing that too much

Reply:
Nah its okay in just one spot I think.